include/wx/msw/dib.h

```diff
@@ -164,7 +164,23 @@ class WXDLLIMPEXP_CORE wxDIB
     bool Create(const wxImage& image, PixelFormat pf = PixelFormat_PreMultiplied);
 
     // create wxImage having the same data as this DIB
+
+    // Possible options of conversion to wxImage
+    enum ConversionFlags
+    {
+        // Determine whether 32bpp DIB contains real alpha channel
+        // and return wxImage with or without alpha channel values.
+        Convert_AlphaAuto,
+        // Assume that 32bpp DIB contains alpha channel and always return
+        // wxImage with alpha channel values in this case.
+        Convert_AlphaAlwaysIf32bpp
+    };
+#if WXWIN_COMPATIBILITY_3_0
```

Comment:
Is there any problem with using `ConvertToImage(ConversionFlags flags = Convert_AlphaAuto)` always? I don't see how is it incompatible with the 3.0 API... Well, actually I do, it would break the existing code taking a pointer to a member function, but this seems so unlikely and is easy to fix (because it would be a compilation error), that I don't think it's really a problem.

So I'd only keep the `#else` branch here.